homeassistant/components/yeelight/binary_sensor.py

```diff
@@ -35,16 +34,6 @@ async def async_added_to_hass(self):
             )
         )
 
-    @property
-    def unique_id(self) -> Optional[str]:
```

Comment:
Removing this property seems to be a breaking change since it will cause the unique id to change for this entity. Is that true?

Reply:
Yes. I forgot to add this back. I'll create a new PR for this.

Reply:
#42562